tests/middleware/test_current_message.py

```diff
@@ -0,0 +1,26 @@
+import remoulade
+from remoulade.middleware import CurrentMessage
+
+
+class TestCurrentMessage:
+    """Class to test the middleware
+    CurrentMessage"""
+
+    def test_middleware_in_actor(self, stub_broker, stub_worker):
+        @remoulade.actor
+        def do_work():
```

Comment:
could you send, like 20 messages, add each of the message id to a list and verify that the message id in the actor and the mesage ids that has been sent are the same ?

Something like
```
        def test_middleware_in_actor(self, stub_broker, stub_worker):
        message_ids = set()

        @remoulade.actor
        def do_work():
            msg = CurrentMessage.get_current_message()
            message_ids.add(msg.message_id)

        stub_broker.add_middleware(CurrentMessage())
        stub_broker.declare_actor(do_work)
        messages = [do_work.send() for _ in range(20)]

        stub_broker.join(do_work.queue_name)
        stub_worker.join()

        assert message_ids == {m.message_id for m in messages}
        # verify no current message outside of actor
        assert CurrentMessage.get_current_message() is None
```

Also the tests with the assert are not working as they raise but inside the actor, so it's caugth.
Do a test a bit like the one I gave but for the second one.